src/configure/configure.ts

```diff
@@ -63,16 +65,34 @@ class PipelineConfigurer {
         await this.getAzureDevOpsDetails();
         await this.getSelectedPipeline();
 
-        if (this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github) {
-            await this.getGitubConnectionService();
-        }
-
         if (!this.inputs.targetResource.resource) {
             await this.getAzureResourceDetails();
         }
+    }
+
+    private async createPreRequisites(): Promise<void> {
+        if (this.inputs.isNewOrganization) {
+            let repoParts = this.inputs.sourceRepository.repositoryName.split("/");
+            this.inputs.projectName = repoParts[repoParts.length-1] || "VSCode-AzurePipelines";
```

Comment:
Lets not name it VSCode-.. Should it just be AzurePipelinesProject
Also  what will repoParts[..] be for different repo types. This seems error prone when we we add new repositories type. can you refactor it such that it gets caught when we add support for new repo provider types